amadeus/referenceData/ShoppingNew.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
```

Comment:
This class is not necessary. All these APIs should be under the class Shopping.cs